rules/cookiewow.json

```diff
@@ -0,0 +1,234 @@
+{
+	"$schema": "../rules.schema.json",
```

Comment:
I actually hand crafted this, with VS Code and JSON schema. VSCode use $schema to point to the schema definition. I'd be open to upload this in a separate PR.

Reply:
If you made a schema for the rules, that would be pretty cool to have. Even just a start that we can build upon.